Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -25,6 +25,11 @@ public abstract class GCloudCmdlet : PSCmdlet, IDisposable
         /// <summary>Placeholder for an unknown cmdlet name when reporting telemetry.</summary>
         private const string UnknownCmdletName = "unknown-cmdlet";
 
+        /// <summary>
+        /// The project that is used by the cmdlet. This value will be used in reporting usage.
+        /// </summary>
+        public virtual string Project { get; set; }
```

Comment:
It is worth calling out a few other things:

- Can it be ignored? What happens if the derived class leaves it as null? What value will be reported to telemetry? Will we fall back to whatever is in the Cloud SDK "active project" value?

- We should explicitly say that "will be converted into hash, and then used in reporting usage". i.e. we want to make clear we don't send project names to Google, and rather anonymized versions of the data. (I believe that is the expected format in the telemetry reporting.)